### ISSUE TITLE
Incorrect bounding rects returned from pygame.draw.aalines

### ISSUE THREAD

**Issue by charlesej** [pygame/pygame#1153]
The following code gives some examples of incorrect bounding rects returned from `pygame.draw.aalines`.

```python
import pygame

WIDTH, HEIGHT = 320, 320
screen = pygame.display.set_mode((WIDTH, HEIGHT))
color = pygame.Color('red')

# Drawing fully on the surface gives an incorrect bounds.
pts = ((0, 0), (WIDTH - 1, HEIGHT - 1))
bounds = pygame.draw.aalines(screen, color, 1, pts, 1)
print('bounds={}, expecting <rect(0, 0, 320, 320)>'.format(bounds))

# Drawing partially off the surface gives an incorrect bounds.
pts = ((-1, -1), (WIDTH - 1, HEIGHT - 1))
bounds = pygame.draw.aalines(screen, color, 1, pts, 1)
print('bounds={}, expecting <rect(0, 0, 320, 320)>'.format(bounds))

# Drawing fully off the surface gives an incorrect bounds.
pts = ((WIDTH + 1, HEIGHT + 1), (WIDTH + 5, HEIGHT + 5))
bounds = pygame.draw.aalines(screen, color, 1, pts, 1)
print('bounds={}, expecting <rect(321, 321, 0, 0)>'.format(bounds))
```

The above code gives the following output. The returned bounding rects are not as expected.
```
pygame 2.0.0.dev2 (SDL 2.0.9, python 3.7.2)
Hello from the pygame community. https://www.pygame.org/contribute.html
bounds=<rect(0, 0, 321, 321)>, expecting <rect(0, 0, 320, 320)>
bounds=<rect(-1, -1, 322, 322)>, expecting <rect(0, 0, 320, 320)>
bounds=<rect(321, 321, 6, 6)>, expecting <rect(321, 321, 0, 0)>
```

System details:
- os: windows 10 (64bit)
- python: 3.7.2 (64bit)
- pygame: 2.0.0.dev2 (SDL: 2.0.9) at c0646b84d47cc8d0dec15deb7029e61932cb0d3e

**Comment by charlesej** [comment]
Test added with @unittest.expectedFailure decorator via #1170.

**Comment by charlesej** [comment]
Resolved by #1494.
